src/components/pages/ShiftSignUp.jsx

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable object-shorthand */
+/* eslint-disable no-unused-vars */
```

Comment:
/* eslint-disable no-unused-vars */

Could you remove this and then all of the unused variables. Its not a big deal, having unused code/variables, but it is still better to only have what we need.